index.html

```diff
@@ -223,5 +223,41 @@ <h2>About<br />Myself</h2>
         </ul>
       </div>
     </section>
+
+    <section class="form-section">
+      <form action="https://formspree.io/f/xkneryal" method="post">
+        <h2>Contact Me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <input
+          type="text"
+          name="name"
+          placeholder="Yeremias"
+          maxlength="30"
+          required
+        />
+
+        <input
+          type="email"
+          name="mail"
+          placeholder="natajayanj14@gmail.com"
+          required
+        />
```

Comment:
- [ ] [OPTIONAL] It will be better if you add a good descriptive placeholder for email and name ex: Your name, Your email.